packages/http-client-csharp/generator/Microsoft.Generator.CSharp/src/Writers/CodeWriter.cs

```diff
@@ -686,11 +700,6 @@ internal CodeWriter WriteDeclaration(string declaration)
 
         public CodeWriter WriteDeclaration(CodeWriterDeclaration declaration)
         {
-            if (_writingXmlDocumentation)
-            {
-                throw new InvalidOperationException("Can't declare variables inside documentation.");
-            }
```

Comment:
I think we need to allow it - because for parameters, they show up in the docs before they show up in the signature